inc/Engine/Optimization/CSSTrait.php

```diff
@@ -207,11 +207,14 @@ private function apply_font_display_swap( $css_file_content ) {
 			'/(?:@font-face)\s*{(?<value>[^}]+)}/',
 			function ( $matches ) {
 				if ( false !== strpos( $matches['value'], 'font-display' ) ) {
-					return $matches[0];
+					preg_match( '/font-display:\s*(?<attrib>\w*);?/', $matches['value'], $attribute );
+					return 'swap' === $attribute['attrib']
```

Comment:
here we can do two changes:
1. Use `strtolower` function so you can check non-case-sensitively.
2. Use descriptive name instead of `attrib` :D, you can use `font_swap_value` for example.

Reply:
So, I've combined the strpos and regex into a single regex eval with the `i` flag. It passes the tests, but please check the new regex with care (tricky!)

Reply:
regex pattern is fine